.github/workflows/ci.yml

```diff
@@ -16,10 +16,8 @@ jobs:
               with:
                   main-branch-name: 'trunk'
             - run: npm ci
-            - run: npm install -g nx@15.8.6
-
-            - run: nx format:check || npx prettier --check .
-            - run: nx affected --target=typecheck
-            - run: nx affected --target=lint --parallel=3
-            - run: nx affected --target=test --parallel=3 --configuration=ci
-            - run: nx affected --target=build --parallel=3
+            - run: node_modules/.bin/nx format:check || npx prettier --check .
+            - run: node_modules/.bin/nx affected --target=typecheck
+            - run: node_modules/.bin/nx affected --target=lint --parallel=3
+            - run: node_modules/.bin/nx affected --target=test --parallel=3 --configuration=ci
+            - run: node_modules/.bin/nx affected --target=build --parallel=3
```

Comment:
does `npx nx` not work here?

Reply:
I’ll try that. Npm + paths issues are somehow too familiar. I wish it was easier to rip out npm than yarn :p